### ISSUE TITLE
Does it work on android 12?

### ISSUE THREAD

**Issue by rew1nter** [Fuzion24/JustTrustMe#67]

**Comment by jpstotz** [comment]
Yes, but the question is more in what environment you are trying to use it. 

The most recent and recommended way for Android 12 would be a Magisk rooted devices in `Zygisk` mode with installed LSPosed in Zygisk mode and JustTrustMe as module installed into LSPosed.

**Comment by gillbates** [comment]
hi @jpstotz , can you advise how to install JustTrustMe as module?

**Comment by jpstotz** [comment]
@gillbates Sorry but I haven't used it for a while. As far as I remember it is a simple apk you install and that then appears in LSPosed.
